spec/integ/crypto/cross-signing.spec.ts

```diff
@@ -86,14 +90,10 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("cross-signing (%s)", (backend: s
      * Create cross-signing keys, publish the keys
      * Mock and bootstrap all the required steps
      *
-     * @return the IAuthDict - The parameters which are submitted as the `auth` dict in a UIA request
+     * @param authDict - The parameters which are submitted as the `auth` dict in a UIA request
      * @see https://spec.matrix.org/v1.6/client-server-api/#authentication-types
      */
-    async function setupCrossSigning(): Promise<IAuthDict> {
-        mockCrossSigningRequest();
-
-        // provide a UIA callback, so that the cross-signing keys are uploaded
-        const authDict = { type: "test" };
+    async function mockCrossSigningRequests(authDict: IAuthDict): Promise<void> {
```

Comment:
this no longer mocks anything.

```suggestion
    async function bootstrapCrossSigning(authDict: IAuthDict): Promise<void> {
```